src/rqt_pose_view/pose_view_widget.py

```diff
@@ -82,6 +82,14 @@ def __init__(self, plugin):
         self._update_timer.timeout.connect(self.update_timeout)
         self._update_timer.start(40)
 
+        # subscribe immediately if topic name found on param server
+        if rospy.has_param('~topic'):
+            topic = rospy.get_param('~topic')
+            try:
+                self._subscribe_topic(topic)
+            except(AttributeError):
+                rospy.logwarn('invalid topic name {}'.format(topic))
```

Comment:
Depending on what the parameter is set to it would be helpful to quote the topic name: `"invalid topic name '{}'"`